static/js/portico/signup.js

```diff
@@ -73,6 +73,15 @@ $(function () {
         $("#timezone").val(moment.tz.guess());
     }
 
+    // Code in this block will be executed when the user visits /register
+    if (window.location.pathname === "/register/") {
+        common.autofocus('#email');
+
+        if (window.location.hash.substring(0, 1) === "#") {
+            document.email_form.action += window.location.hash;
```

Comment:
@timabbott I think we should discuss this line in context to ensuring users land on the narrow they actually visited even if they had to undergo registration. So I feel like this line is from 2013 code base and have never really worked. Basic reasoning for this is that when users register they get a confirmation email from which they again click a link and proceed with registration before ending up in the UI. Thing is we never carry forward the narrow to that registration email. Now the expected behaviour around that verification mail is a bit fuzzy. Either we should  pass on this narrow link to that email or keep the current behaviour of forgetting what narrow user originally visited.

Reply:
I think this change to check by `/register/` as the path is potentially buggy, since according to zproject/urls.py, `accounts_home` is also visible at the `/accounts/home/` path.  I ... have no idea why that is -- might be some legacy bug that we need to cleanup.